src/components/Input/index.tsx

```diff
@@ -0,0 +1,7 @@
+import style from '@/components/Input/index.module.css';
+
+const Input = () => {
+  return <input className={style.groupNameInput}></input>;
```

Comment:
<input> 안에 들어갈 것이 없다면 self closing 으로 수정하는게 좋을 것 같습니다
```suggestion
  return <input className={style.groupNameInput} />;
```

Reply:
[f5fd44f](https://github.com/sjoleee/bemyidol-frontend/pull/19/commits/f5fd44f9c620c9d3f8c3a7f648ede454c4fdd126)에서 반영했습니다!